docs/sources/reference/components/otelcol.receiver.file_stats.md

```diff
@@ -0,0 +1,269 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol.receiver.file_stats/
+title: otelcol.receiver.file_stats
+description: Learn about otelcol.receiver.file_stats
+---
+
+# otelcol.receiver.file_stats
+
+`otelcol.receiver.file_stats` collects metrics from files specified with a glob pattern.
```

Comment:
Does this only do files? Several of the metrics mention folders

Reply:
it does both, I'll make sure I mention files and folders more consistently